sdk/python/feast/templates/aws/feature_repo/feature_store.yaml

```diff
@@ -0,0 +1,27 @@
+project: my_project
+# By default, the registry is a file (but can be turned into a more scalable SQL-backed registry)
+# On AWS, minimally you should create a S3 bucket for a remote file registry
+registry: data/registry.db
```

Comment:
should the default be using an s3 bucket? or at least should there be an example of how to use an s3 bucket (e.g. commented line)?